src/media/css/header.styl

```diff
@@ -448,3 +448,12 @@ body > .banners mkt-banner {
         background: url(../img/pretty/marketplace-beta-logo2X.png) no-repeat 0 50% / 200px auto;
     }
 }
+
+@media $base-desktop {
```

Comment:
Your earlier fix of `padding: 40px 10px 0` was better but it needs to be within `@media $base-tablet` and not apply to the `a`. Also `0` values don't need unit types such as `px` specified.